src/blocks/Slider/Slider.tsx

```diff
@@ -251,7 +251,7 @@ export const SliderBlock = (props: WithChildren<SliderProps>) => {
         return (
             // To have this key differ from keys used in renderDot function, added `-accessible-bar` part
             <Fragment key={`${index}-accessible-bar`}>
-                {slidesCountByBreakpoint > 1 && (
+                {slidesCountByBreakpoint > 0 && (
```

Comment:
why did you change the value? Could it break something?

Reply:
I change it because when we have `slidesCountByBreakpoint === 1` the accessible bar didn't render and one was unable to find it vis screenreader